ansible/roles/software/ckan/inventory/molecule/default/molecule.yml

```diff
@@ -35,7 +35,7 @@ provisioner:
         inventory_ckan_envs: |
           TEST_ENV=1
       bionic:
-        inventory_app_repo_branch: inventory_ckan_2.8
```

Comment:
I think for now we should leave this as inventory_ckan_2.8 since that branch will only be targeting FCS and not cloud.gov.

@FuhuXia @jbrown-xentity what do you think the right thing to do is? Will we merge inventory_ckan_2.8 into `fcs` or will we just replace the `fcs` branch at launch time?

Reply:
I think we would just point to the new branch, since it will only be that way for a short time (moving to either `main` or `master` for cloud.gov)